plugins/chat/assets/stylesheets/common/base-common.scss

```diff
@@ -267,6 +267,7 @@ html.has-full-page-chat {
       .main-chat-outlet {
         min-height: 0;
         max-width: 100vw;
+        box-sizing: border-box;
```

Comment:
These border-box changes are only slightly related, but they avoid some overflows in panels